events/event/models.py

```diff
@@ -2,15 +2,24 @@
 from registration.models import User
 from django.db.models.fields.related import ManyToManyField
 from django.contrib.postgres.fields import ArrayField
+from django.utils.timezone import utc
+
+from datetime import datetime
+
+
+
+TZ = utc
```

Comment:
But you can use directly `utc` without redefining it.  I mean `from django.utils.timezone import utc as TZ`. Will it work?